### PR TITLE
Add talks to schedule page

### PR DESCRIPTION
**Pull request by crackofdusk**
This may not be the final version...

### CODE REVIEW

File: src/2023/schedule/index.html [hunk 1]
```diff
@@ -41,6 +41,54 @@ <h2 class="schedule-item-title">Registration</h2>
             <h2 class="schedule-item-title">Opening words</h2>
```
Comment (hharen) [inline src/2023/schedule/index.html:41]:
```suggestion
            <h2 class="schedule-item-title">Start of the conference - opening words</h2>
```

Comment (hharen) [reply src/2023/schedule/index.html:41]: I'd add this to make it clear that people should be there and seated at 9:30. As "Opening words" can be easily misread as "Opening doors" and people may do this mistake and come just after 9:45.

File: src/2023/schedule/index.html [hunk 2]
```diff
@@ -41,6 +41,54 @@ <h2 class="schedule-item-title">Registration</h2>
...
+              <p class="schedule-details">Abiodun Olowode</p>
+            </div>
+          </article>
+
+          <article class="schedule-item">
+            <time class="schedule-time">11:20</time>
+            <div>
+              <h2 class="schedule-item-title">
+                <a href="../talks#karen-jex-how-to-keep-your-database-happy">
+                  How to Keep your Database Happy
```
Comment (hharen) [inline src/2023/schedule/index.html:73]:
```suggestion
                  How to Keep Your Database Happy
```

wdyt?

File: src/2023/schedule/index.html (hunk 1 above)
Comment (lxxxvi) [reply src/2023/schedule/index.html:41]: I don't have a strong opinion on this, but I think the schedule item above "Registration" makes it clear enough that "Opening words" is not "Opening doors". 🤷

**Review by lxxxvi — APPROVED** [review]
Let's go with this... we could add a note somewhere saying something like "Subject to change".

**Comment by crackofdusk** [comment]
> We have time slots of 20, 30 and 40 minutes. I thought we'd have just 20 and 30?

All slots are 20-30 minutes, but there are shorter and longer breaks. I did not add any explicit breaks to the schedule except the lunch break.

**Comment by crackofdusk** [comment]
> we could add a note somewhere saying something like "Subject to change"

The exact timing and sequence probably matters the most to the speakers.

For the rest, I think we'll be alright as long as we keep showing the schedule during the day of the conference.

**Comment by crackofdusk** [comment]
Let's open a different pull request for capitalisation changes. These need to happen in multiple places.

**Pull request merged**